package.json

```diff
@@ -48,6 +48,6 @@
     "tape": "^4.11.0"
   },
   "engines": {
-    "node": ">= 6"
+    "node": ">= 7"
```

Comment:
Node 6 was already unsupported, because `async`/`await` was being used. Thus, there's no need for a major version bump here.

Reply:
Node 6 is still supported using Babel: https://github.com/terkelg/prompts/blob/master/index.js

Reply:
Ah, I didn't realize that. Ok. Well, Node 6 is EOL in any case. How would you feel about dropping support for it? My use of `async` / `await` will still require a bit more dev work to make the tests pass in Node 6.

Reply:
It's a good idea to drop Node 6 support, but I don't think we should do that before v3

Reply:
Very thought out PR. Thank you for paying attention to things like versioning. - appreciate it. However I have a script that automatically bump version and publish. Would you mind reverting this?

Reply:
Yeah, for sure dropping Node 6 support should be behind a major version bump. And I take it you don't want this PR to be v3? :smile: I'll revert.